dotnet/Service/Program.cs

```diff
@@ -132,7 +133,7 @@
                 ILogger<Program> log) =>
             {
                 log.LogTrace("New search request");
-                MemoryAnswer answer = await service.AskAsync(question: query.Question, index: query.Index, query.Filter);
+                MemoryAnswer answer = await service.AskAsync(question: query.Question, index: query.Index, filters: new List<MemoryFilter>() { query.Filter });
```

Comment:
pls update the `MemoryQuery` class to support the same list of filters

Reply:
Done.